src/main/java/levy/dictionary/EnglishDictionary.java

```diff
@@ -0,0 +1,81 @@
+package levy.dictionary;
+
+import com.opencsv.CSVReader;
+import com.opencsv.exceptions.CsvValidationException;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+public class EnglishDictionary {
+
+    private Map<String, List<Definition>> dictionary;
+
+    public EnglishDictionary() throws CsvValidationException, IOException {
+        dictionary = new HashMap<>();
+
+        // Load data from CSV file
+        try (InputStream in = getClass().getResourceAsStream("/englishDictionary.csv");
+             CSVReader reader = new CSVReader(new InputStreamReader(in))) {
+
+            String[] record;
+
+            while ((record = reader.readNext()) != null) {
+                if (record.length >= 2) {
+                    String word = record[0].trim().toLowerCase();
+                    String[] parts = record[1].split(",");
+                    String partOfSpeech = parts[0].trim();
+                    String definition = record[1].substring(record[1].indexOf(",") + 1).trim();
```

Comment:
What is this for?

Reply:
just more organized when i was researching/learning how to do i saw a lot of this